_cloudstor/01-overview.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Overview
+order: 1
+duration: 5
+---
+
+### What is AARNET CloudStor
+
+CloudStor is an AARNet developed and supported service that enables AARNet customers and the wider community to quickly and securely sync, share and store files using the high-speed AARNet network. AARnet CloudStor provides a easy to use web based dashboard and You can login the dashboard via [Cloudstor dashboard](https://cloudstor.aarnet.edu.au ) and it will give the user access to both CloudStor (storage) and FileSender (ideal for sharing large files). For information about how to use the dashboard, please refer to [CloudStor Getting Started Guide](https://support.aarnet.edu.au/hc/en-us/articles/227469547-CloudStor-Getting-Started-Guide).
+ 
+ Nectar offers a helper command called `cloudstor-setup` for users who wish to make their CloudStor storage available on their Nectar virtual machine.
+ 
+ In this tutoria, you will walk through the setup procedure, and mounts your CloudStor storage to /cloustor within your virtual machine.
+
+**Cloud Starter**
+This tutorial is part of the Nectar Cloud Starter curriculum. You should be familiar with the absolute basics of Security Groups, you should have a Keypair and you should have terminal software installed on your machine. If you think you need help with any of that, you should complete Cloud Starter tutorials before you start here.
+{: .callout-warning}
+
+### What you'll learn
+
+- Mount CloudStor storage to /cloustor with your virtual machine
+
+### What you'll need
+
+- A running virtual machine
+- [Terminal software](https://support.ehelp.edu.au/support/solutions/articles/6000223964-terminal-software)
+- Access to a Nectar Project
```

Comment:
You would have this if you have a running virtual machine. Maybe revise the requirements here